microsoft-365/security/defender-endpoint/mac-jamfpro-policies.md

```diff
@@ -414,12 +414,12 @@ These steps are applicable of macOS 10.15 (Catalina) or newer.
         :::image type="content" source="images/7f9138053dbcbf928e5182ee7b295ebe.png" alt-text="The configuration settings mdatpmdav notifications tray" lightbox="images/7f9138053dbcbf928e5182ee7b295ebe.png":::
 
     - Tab **Notifications**, click **Add** one more time, scroll down to **New Notifications Settings**
-        - **Bundle ID**: `com.microsoft.autoupdate2`
+        - **Bundle ID**: `com.microsoft.autoupdate.fba`
         - Configure the rest of the settings to the same values as above
 
         :::image type="content" source="images/4bac6ce277aedfb4a674f2d9fcb2599a.png" alt-text="The configuration settings mdatpmdav notifications mau" lightbox="images/4bac6ce277aedfb4a674f2d9fcb2599a.png":::
```

Comment:
Ideally this image would also be updated, but probably best to be done by MS, so as to provide consistency.